src/Package/Impl/Help/SearchWebForCurrentCommand.cs

```diff
@@ -32,11 +36,26 @@ internal sealed class SearchWebForCurrentCommand : HelpOnCurrentCommandBase {
 
         protected override void Handle(string item) {
             // Bing: search?q=item+site%3Astackoverflow.com
-            var encoded = HttpUtility.HtmlEncode(item);
-            var search = "http://" + Invariant($"www.bing.com/search?q={encoded}+R+site%3A{RToolsSettings.Current.HelpSearchSite}");
-            var browser = VsAppShell.Current.GetGlobalService<IVsWebBrowsingService>(typeof(SVsWebBrowsingService));
-            IVsWindowFrame frame;
-            browser.Navigate(search, (uint)__VSWBNAVIGATEFLAGS.VSNWB_WebURLOnly, out frame);
+            var tokens = RToolsSettings.Current.WebHelpSearchString.Split(new char[] { ' ', '\t' }, StringSplitOptions.RemoveEmptyEntries);
+            bool siteSet = false;
+
+            var sb = new StringBuilder("http://" + Invariant($"www.bing.com/search?q={HttpUtility.HtmlEncode(item)}"));
+            foreach (var t in tokens) {
+                sb.Append('+');
+                if (!siteSet && t.IndexOf('.') > 0) {
+                    sb.Append("site:");
```

Comment:
I'd rather let them specify it explicitly if they want site: or not. Not everything that contains a dot is necessarily a site.